src/recipes.jl

```diff
@@ -7,18 +7,24 @@ export partitionplot
     args
 end
 
-@recipe function f(o::PartitionPlot; nparts = 100, stat = nothing, by = nothing)
+getvalue(x) = x 
+getvalue(x::DataValues.DataValue) = get(x)
+
+@recipe function f(o::PartitionPlot; nparts = 100, stat = nothing, by = nothing, dropmissing = false)
     t = o.args[1]
     sel_x = o.args[2] 
     stat = (stat == nothing) ? Extrema() : stat
     if length(o.args) == 3
         sel_y = o.args[3]
         T = fieldtype(eltype(t), IndexedTables.colindex(t, sel_x))
+        s = dropmissing ? 
+            series(IndexedPartition(T, stat, nparts)) :
+            series(IndexedPartition(T, stat, nparts); filter = !isnull, transform = getvalue)
```

Comment:
Don't you need to also take care of the case where you select two variables? Shouldn't it be something like `transform = v -> map(getvalue, v)` and `filter = v -> !any(isnull, v)` in that case?

Reply:
Yep, I also swapped the logic for when to apply the filter/transform.  Fixed now.